test/unit/app/azure-storage/query-entities.test.js

```diff
@@ -49,70 +43,51 @@ describe('queryEntitiesByPartitionKey', () => {
       when: {
       },
       expect: {
-        events: [],
-        consoleLogs: [
-          `${MOCK_NOW.toISOString()} Listing events by: ${JSON.stringify({
-            tableName: 'table_name',
-            partitionKey: ''
-          })}`
-        ]
+        events: []
       }
     },
     {
       toString: () => 'No events found',
       given: {
         tableName: 'table_name',
-        partitionKey: 'partition_key'
+        partitionKey: 123456789
       },
       when: {
         entities: []
       },
       expect: {
-        events: [],
-        consoleLogs: [
-          `${MOCK_NOW.toISOString()} Listing events by: ${JSON.stringify({
-            tableName: 'table_name',
-            partitionKey: 'partition_key'
-          })}`,
-          `${MOCK_NOW.toISOString()} Creating the table client using the DefaultAzureCredential: ${JSON.stringify({
-            tableName: 'table_name'
-          })}`
-        ]
+        events: []
       }
     },
     {
       toString: () => 'One event found',
       given: {
         tableName: 'table_name',
-        partitionKey: 'partition_key'
+        partitionKey: 123456789
       },
       when: {
```

Comment:
If I'm reading this test correctly, it's returning an empty array not a single event. Could we also add a test where the partitionKey in the table is `123456789_`